pkgs/development/python-modules/asyncsleepiq/default.nix

```
@@ -7,14 +7,14 @@
 
 buildPythonPackage rec {
   pname = "asyncsleepiq";
-  version = "1.4.0";
+  version = "1.4.1";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and put `setuptools` in `nativeBuildInputs`.

---

Thanks, updated.